cmd/plugin/cli/upgrade.go

```diff
@@ -41,6 +41,7 @@ func UpgradeCmd() *cobra.Command {
 			return nil
 		},
 	}
+	cmd.Flags().Bool(installer.WaitFlag, false, "wait for operation finish")
```

Comment:
Nit: I think this should be a bit more descriptive, maybe:
 `wait for storagos cluster to enter running phase` ?